### PR TITLE
control/new: add new appearance types for media inputs.

### PR DESCRIPTION
**Pull request by clint-tseng**

### CODE REVIEW

File: public/javascripts/data.js [hunk 1]
```diff
@@ -148,6 +148,23 @@ var dataNS = odkmaker.namespace.load('odkmaker.data');
...
     };
+    var mediaTypes = {
+         'Image': 'image/*',
+         'Selfie': 'image/*',
+         'Annotate': 'image/*',
+         'Draw': 'image/*',
+         'Signature': 'image/*',
+         'Audio': 'audio/*',
+         'Video': 'video/*',
+         'Selfie Video': 'video/*'
```
Comment (yanokwa) [inline public/javascripts/data.js]: There's no `Selfie Video` in Collect that I'm aware of. Where'd ya find it?

File: public/javascripts/data.js [hunk 2]
```diff
@@ -148,6 +148,23 @@ var dataNS = odkmaker.namespace.load('odkmaker.data');
         'Vertical Slider': 'vertical',
         'Picker': 'picker'
     };
+    var mediaTypes = {
```
Comment (yanokwa) [inline public/javascripts/data.js]: If we are adding media types, the `new` appearance is something folks want. 
https://docs.opendatakit.org/form-question-types/#image-widget-with-required-new-image

File: public/javascripts/data.js (hunk 1 above)
Comment (clint-tseng) [reply public/javascripts/data.js]: https://github.com/opendatakit/build/issues/200

File: public/javascripts/data.js (hunk 2 above)
Comment (clint-tseng) [reply public/javascripts/data.js]: okay

File: public/javascripts/data.js (hunk 1 above)
Comment (yanokwa) [reply public/javascripts/data.js]: Well, I'll be damned. Checked in the source, and it's definitely there, but it isn't in the docs. I've filed it at https://github.com/opendatakit/docs/issues/982.

In the future, can you link to the issue (if it exists) in the PR. It'd prevent me from embarrassing myself.

File: public/javascripts/data.js [hunk 3]
```diff
@@ -148,6 +148,23 @@ var dataNS = odkmaker.namespace.load('odkmaker.data');
...
+         'Audio': 'audio/*',
+         'Video': 'video/*',
+         'Selfie Video': 'video/*'
+    };
+    var mediaAppearances = {
+          'Signature': 'signature',
+          'Annotate': 'annotate',
+          'Draw': 'draw',
+          'Selfie': 'new-front',
+          'Selfie Video': 'new-front'
```
Comment (yanokwa) [inline public/javascripts/data.js]: new-front is correct for selfie-video. @lognaturel had a copy and paste error in the issue description.

File: public/javascripts/data.js (hunk 1 above)
Comment (clint-tseng) [reply public/javascripts/data.js]: sorry, it's usually tagged in the commit message but apparently i forgot this time. it _is_ the name of the branch though :)

**Comment by clint-tseng** [comment]
@yanokwa added `new`

**Pull request merged**